### PR TITLE
Test bench components with more values

### PR DESCRIPTION
**Pull request by ggwpez**
Currently the benchmarks are executed in test mode with the lowest and highest value of each component. This lead to some cases in the past where the `cargo test` passed but the `cargo run` did not.   
For example the range `0 .. 10` is tested with the values `[0, 10]`.  
This MR changes it to six values in total, so it becomes `[0, 2, 4, 6, 8, 10]`.

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

File: frame/benchmarking/src/lib.rs [hunk 1]
```diff
@@ -1195,16 +1195,28 @@ macro_rules! impl_benchmark_test {
...
-							// Test only the low and high value, assuming values in the middle
-							// won't break
-							for component_value in $crate::vec![low, high] {
+						for (name, low, high) in components.clone().into_iter() {
+							// Test the lowest, highest (if its different from the lowest)
+							// and up to 4 more equidistant values in between.
+							// For 0 .. 10 this would mean: [0, 2, 4, 6, 8, 10]
+
+							let mut values = $crate::vec![low];
+							let diff = (high - low).min(5);
```
Comment (ggwpez) [inline frame/benchmarking/src/lib.rs:1204]: TODO add env var

**Comment by shawntabrizi** [comment]
Can you add an environment variable, and keep this as the default

File: frame/benchmarking/src/lib.rs (hunk 1 above)
Comment (ggwpez) [reply frame/benchmarking/src/lib.rs]: Updated it @shawntabrizi

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**